Add KiCad project file scrub (#14)
apcountryman/prototype-kicad-library-ci: README.md

`prototype-kicad-library-ci` uses the [GitHub
flow](https://guides.github.com/introduction/flow/) workflow.

> ## Git Config Install Instructions

Reviewer: Please change the subsection title to `Git Config` (as was requested in #13) for consistency with the `Git Hooks` subsection title.
```suggestion
## Git Config
```

> To install this repository's Git configuration, execute the `install` script located in the

Reviewer: Please change `Git configuration` to `Git config entries` which more accurately describes what the `git/config/install` script does (add entries to the repository's `.git/config` file), and matches the script's comment block.
```suggestion
To install this repository's Git config entries, execute the `install` script located in the
```

Please use a column limit of 90 (see https://github.com/apcountryman/prototype-kicad-library-ci/blob/main/CONTRIBUTING.md#style-guide). Couldn't include this in the suggested change due to a possible bug in the raw markdown viewer (can't select multiple lines).

`prototype-kicad-library-ci` uses the [GitHub
flow](https://guides.github.com/introduction/flow/) workflow.

## Git Config
To install this repository's Git config entries, execute the `install` script located in the

> `git/config` directory.

Reviewer: Please use a column limit of 90 (see https://github.com/apcountryman/prototype-kicad-library-ci/blob/main/CONTRIBUTING.md#style-guide), as was noted in https://github.com/apcountryman/prototype-kicad-library-ci/pull/14#discussion_r644434877.
```suggestion
To install this repository's Git config entries, execute the `install` script located in
the `git/config` directory.
```
